packages/babel-parser/src/parse-error.ts

```diff
@@ -103,23 +112,34 @@ function toParseErrorConstructor<ErrorDetails: Object>({
 // to the type system, avoiding the need to do so with $ObjMap (which doesn't
 // work) in `ParseErrorEnum`. This hack won't be necessary when we switch to
 // Typescript.
-declare function toParseErrorCredentials<T: string>(
-  T,
-  ?{ code?: ParseErrorCode, reasonCode?: string } | boolean,
-): ParseErrorConstructor<{||}>;
-
-// ESLint seems to erroneously think that Flow's overloading syntax is an
-// accidental redeclaration of the function:
-// https://github.com/babel/eslint-plugin-babel/issues/162
-// eslint-disable-next-line no-redeclare
-declare function toParseErrorCredentials<ErrorDetails>(
-  (ErrorDetails) => string,
-  ?{ code?: ParseErrorCode, reasonCode?: string } | boolean,
+function toParseErrorCredentials(
+  b: string,
+  a?:
```

Comment:
`export` was added before the comment, which seems to be a problem with the automatic conversion.